OpenOversight/app/templates/about.html

```diff
@@ -1,150 +1,158 @@
 {% extends "base.html" %}
+
 {% block title %}About OpenOversight{% endblock %}
+
 {% block meta %}
-<meta name="description" content="OpenOversight is a Lucy Parsons Labs project that aims to improve law enforcement
+<meta
+  name="description"
+  content="OpenOversight is a Seattle Tech Bloc project that aims to improve law enforcement
 visibility and transparency using public and crowdsourced data. We maintain databases, digital galleries, and profiles of individual law enforcement officers from departments
-across the United States that consolidate information including names, birthdates, mentions in news articles, salaries, and photographs.">
+the so called Pacific Northwest that consolidate information including names, birthdates, mentions in news articles, salaries, and photographs."
+/>
 {% endblock %}
+
 {% block content %}
 
 <div role="main">
-<div class="hero-section no-sub">
-  <div class="hero">About OpenOversight</div>
-</div>
-<div class="vertical-padding">
-  <p class="lead"><small>
-    OpenOversight is a <a href="https://lucyparsonslabs.com">Lucy Parsons Labs</a> project that aims to improve law enforcement
-    visibility and transparency using public and crowdsourced data. We maintain databases, digital galleries, and profiles of individual law enforcement officers from departments
-    across the United States that consolidate information including names, birthdates, mentions in news articles, salaries, and photographs.
-  </small></p>
-
-  <div class="text-center pb-50"><a href="/find" class="btn btn-lg btn-info" id="try-it">
-    Try it
-  </a></div>
+  <div class="hero-section no-sub">
+    <div class="hero">About OpenOversight</div>
+  </div>
+  <div class="vertical-padding">
+    <p class="lead">
+      <small>
+        OpenOversight is a
+        <a
+          href="https://lucyparsonslabs.com"
+          rel="noopener noreferrer"
+          target="_blank"
+          >Lucy Parsons Labs</a
+        >
+        project that aims to improve law enforcement visibility and transparency
+        using public and crowdsourced data. We maintain databases, digital
+        galleries, and profiles of individual law enforcement officers from
+        departments across the United States that consolidate information
```

Comment:
Did we want to change this to match the meta tag above?

Reply:
Oh definitely. Done in 4a4f498